### PR TITLE
[r+] Doc fixes

### PR DESCRIPTION
**Pull request by alkor**
Corrections in TRPL macros guide

### CODE REVIEW

File: src/librustc/metadata/encoder.rs [hunk 1]
```diff
@@ -2035,7 +2035,7 @@ pub fn encode_metadata(parms: EncodeParams, krate: &ast::Crate) -> Vec<u8> {
 
     // And here we run into yet another obscure archive bug: in which metadata
     // loaded from archives may have trailing garbage bytes. Awhile back one of
-    // our tests was failing sporadically on the OSX 64-bit builders (both nopt
+    // our tests were failing sporadically on the OSX 64-bit builders (both nopt
```
Comment (huonw) [inline src/librustc/metadata/encoder.rs]: I think this was correct "one test was failing"

**Comment by alkor** [comment]
Oops, there was a wrong patch, reverted it

**Comment by steveklabnik** [comment]
This looks good to me, but can you squash into one commit, please? r=me after

**Comment by alkor** [comment]
r= @steveklabnik

Squashed into one commit

**Comment by steveklabnik** [comment]
@alkor it only lets people with permission do it, and you have to put it on the commit, rather than the PR. But thanks!

**Comment by alkor** [comment]
@steveklabnik Where can I find info about these r[-+=] commands? Readme at https://github.com/graydon/bors seems incomplete and I only briefly looked at the source code of bors.
Sorry, I should have asked this question earlier instead of trying to guess how the system works.

**Comment by steveklabnik** [comment]
@alkor https://github.com/rust-lang/rust/wiki/Note-bors-usage

**Comment by alkor** [comment]
@steveklabnik Thank you!

**Pull request merged**